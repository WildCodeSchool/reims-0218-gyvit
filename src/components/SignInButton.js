import React from "react"
import { Button } from "reactstrap"

const SignInButton = () => (
  <div>
    <Button color="info">Sign in</Button>
  </div>
)

export default SignInButton
