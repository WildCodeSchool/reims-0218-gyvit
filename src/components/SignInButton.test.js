//TODO CREATE TEST

import React from "react"
import ReactDOM from "react-dom"
import renderer from "react-test-renderer"
import SignInButton from "./SignInButton"

it("renders without crashing", () => {
  const div = document.createElement("div")
  ReactDOM.render(<SignInButton />, div)
  ReactDOM.unmountComponentAtNode(div)
})

it("renders link html", () => {
  const tree = renderer.create(<SignInButton />).toJSON()
  expect(tree).toMatchSnapshot()
})
