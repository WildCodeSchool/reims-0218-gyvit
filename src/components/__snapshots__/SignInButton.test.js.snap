// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders link html 1`] = `
<div>
  <button
    className="btn btn-info"
    onClick={[Function]}
    type={undefined}
  >
    Sign in
  </button>
</div>
`;
